Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp

```diff
@@ -467,12 +480,22 @@ Scene_polyhedron_item::compute_normals_and_vertices(void) const
                 positions_facets.push_back(p.y());
                 positions_facets.push_back(p.z());
                 positions_facets.push_back(1.0);
+                vertices[i][0]= p.x();
+                vertices[i][1]= p.y();
+                vertices[i][2]= p.z();
+                i = (i+1) %3;
             }
+            if((vertices[0][0] == vertices[1][0] && vertices[0][1] == vertices[1][1] && vertices[0][2] == vertices[1][2])
+            || (vertices[0][0] == vertices[2][0] && vertices[0][1] == vertices[2][1] && vertices[0][2] == vertices[2][2])
+            || (vertices[2][0] == vertices[1][0] && vertices[2][1] == vertices[1][1] && vertices[2][2] == vertices[1][2]))
+                number_of_degenerated_faces++;
+
         }
```

Comment:
I'll just use the PMP internal function "is_degenerated" that jane just showed me.